ui/jobs/jobs_config.ts

```diff
@@ -238,7 +238,7 @@ UserConfig.registerOptions('jobs', {
           'Wenn der Countdown nahe 0 ist.': 'threshold',
         },
       },
-      default: 'disabled',
+      default: 'expired',
```

Comment:
Should the default be 'threshold' instead? Normally you want to reapply e.g. a dot before it falls off.

Reply:
Good idea, thanks!